docs/_articles/en/v3.x/salesforce-graph-engine/rules.md

```diff
@@ -207,61 +248,27 @@ Match any violation message that you receive with these cases to understand more
 
 #### Loop Case
 
-`Schema.getGlobalDescribe` was called inside a ForLoopStatement at <class>:<line>.
-`Schema.describeSObjects` was called inside a WhileLoopStatement at <class>:<line>.  
+`Schema.getGlobalDescribe` was called inside a loop. `[ForEachStatement at AuraEnabledFls:27]`
```

Comment:
Should this be preceded by a `>` so it renders as a box the way the ones for other rules do?